presto-tests/src/test/java/com/facebook/presto/tests/TestTpchLocalStats.java

```diff
@@ -71,4 +71,14 @@ public void testTableScanStats()
                         .verifyExactColumnStatistics("n_regionkey")
                         .verifyExactColumnStatistics("n_name"));
     }
+
+    @Test
+    public void testDateComparisons()
+    {
+        statisticsAssertion.check("SELECT * FROM orders WHERE o_orderdate >= DATE '1993-10-01'",
```

Comment:
what about `=, !=` and symbol vs symbol?

Reply:
symbol vs symbol is not supported. `=`, `!=` are not universally interesting (deviation in the tests depends on actual frequency of the value chosen for the test).